sb3_contrib/common/wrappers/time_feature.py

```diff
@@ -0,0 +1,88 @@
+from typing import Dict, Union
+
+import gym
+import numpy as np
+from stable_baselines3.common.type_aliases import GymObs, GymStepReturn
+
+
+class TimeFeatureWrapper(gym.Wrapper):
+    """
+    Add remaining time to observation space for fixed length episodes.
+    See https://arxiv.org/abs/1712.00378 and https://github.com/aravindr93/mjrl/issues/13.
+
+    .. note::
+
+        Only ``gym.spaces.Box`` and ``gym.spaces.Dict`` (``gym.GoalEnv``) observation spaces
+        are supported for now.
```

Comment:
I'd also emphasize that only 1D observations are supported and also include check in the init (`assert len(env.observation_space.shape) == 1`).